src/Network/Network/Generated/Models/PSExpressRouteLink.cs

```diff
@@ -53,5 +53,13 @@ public partial class PSExpressRouteLink : PSChildResource
         [JsonProperty(Order = 1)]
         [Ps1Xml(Target = ViewControl.Table)]
         public string ProvisioningState { get; set; }
+        [JsonProperty(Order = 1)]
+        public ExpressRouteLinkMacSecConfig MacSecConfig { get; set; }
```

Comment:
PowerShell model type should start with `PS`, also you seem to be missing this model

Reply:
Done changed it to PSExpressRouteLinkMacSecConfig